### DISCUSSION TITLE
An uninformed question on the differences between this and using the main model

### DISCUSSION THREAD

**Discussion by Jamesjjvr** [acheong08/ChatGPT#409] [Q&A] [answered]
ChatGPT to my knowledge and experience has a "word limit" of about 4 thousand words before it begins to forget past information. Is this limit bypassed through this method or is the limit baked into the AI. I'm assuming not, but its better to ask.

Thank you for your time!

**Comment by acheong08** [comment, marked as answer]
It's a limit baked in to the model. You can ask it summarize all previous content and send that so that it at least remembers some previous context before it runs out of memory
